### DISCUSSION TITLE
Adding percentile calculations for other ancestry pops

### DISCUSSION THREAD

**Discussion by h171e** [PGScatalog/pgsc_calc#218] [Q&A] [answered]
Is it possible to obtain additional percentile scores calculations for other pops in the reference and for all of the pops in the reference? If so how can these options be specified?

**Comment by smlmbrt** [comment, marked as answer]
Hi @h171e, no not at the moment. However, the output files (`[sampleset]_popsimilarity.txt.gz` and `[sampleset]_pgs.txt.gz`) from the genetic ancestry analysis have all the information you need to do this (see https://pgsc-calc.readthedocs.io/en/latest/explanation/geneticancestry.html and https://pgsc-calc.readthedocs.io/en/latest/explanation/output.html). Basically the PGS for the reference samples are in the PGS file, and the superpopulation and relatedness information are in the popsimilarity file. You can use those to calculate percentiles and see how we do those calculations in the code here: https://github.com/PGScatalog/pgscatalog_utils/blob/6da7eb0e157ba4e73f941233ee8d8ae4fb5e3926/pgscatalog_utils/ancestry/tools.py#L255-L276
